cocos/base/ccUtils.h

```diff
@@ -64,7 +65,16 @@ namespace utils
      * base filename ("hello.png" etc.), don't use a relative path containing directory names.("mydir/hello.png" etc.).
      * @since v3.2
      */
-    void CC_DLL captureScreen(const std::function<void(bool, const std::string&)>& afterCaptured, const std::string& filename);
+    CC_DLL void  captureScreen(const std::function<void(bool, const std::string&)>& afterCaptured, const std::string& filename);
+
+    /** Capture a specific Node.
+    * @param startNode: specify the snapshot Node. It chould be cocos2d::Scene
+    * @param scale
+    * @returns: return a Image, then can call saveToFile to save the image as "xxx.png or xxx.jpg".
+    * @since v3.11
+    * !!! remark: Caller is responsible for releasing it by calling delete.
+    */
+    CC_DLL Image* captureNode(Node* startNode, float scale = 1.0f);
```

Comment:
You mean, Let me modify the captureScreen implemention? The captureNode is finished snapshot at this frame. If call this function at captureScreen internally, "captureScreen" prototype will be void captureScreen(const std::string& filename), the call back "const std::function<void(bool, const std::string&)>& afterCaptured" never needed. such as follow:

void captureScreen(std::string& filename)
{
     auto image = captureNode(Director::getInstance()->getRunningScene());
    if (FileUtils::getInstance()->isAbsolutePath(filename))
            {
                outputFile = filename;
            }
            else
            {
                CCASSERT(filename.find("/") == std::string::npos, "The existence of a relative path is not guaranteed!");
                outputFile = FileUtils::getInstance()->getWritablePath() + filename;
            }
     AsyncTaskPool::getInstance()->enqueue(AsyncTaskPool::TaskType::TASK_IO, mainThread, (void*)NULL, [image, outputFile]()
            {
                succeedSaveToFile = image->saveToFile(outputFile);
                image->release();
            });
}